characters.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Character of Kaamelott to retrieve sentences from
+type Character struct {
+	Sentences []string `json:"sentences"`
+}
+
+func loadCharacters() {
+	for index := range Configuration.Characters.List {

```

Comment:
it may be interesting to rely on a list of file available in the folder to make it easier to add a character by just adding the file

Reply:
The improvement is implemented. It now loads all the files in the folder defined in the properties and add them to a character list.